builtin/mainmenu/tab_online.lua

```diff
@@ -173,7 +174,7 @@ local function get_formspec(tabview, name, tabdata)
 		retval = retval .. ";0]"
 	end
 
-	return retval, "size[15.5,7,false]real_coordinates[true]"
+	return retval, "size[15.5,7.95,false]real_coordinates[true]"
```

Comment:
Tabs shouldn't be different sizes. There are two solutions I see

## 1: Icons

```
★ [Rename] [Connect]
```

or a bin icon. With a star icon, you could also allow users to add favourites.

## 2: Steal from description textarea height

You should also swap connect and rename, and right align, to be consistent with join game like so:

```
[Del. Favourite] [Rename]
                [Connect]
````

Reply:
I understand the "steal from description textarea height". Would you rather that, or resize the other tabs to match the Join Game tab?

By "icons", do you mean use icons instead of text buttons?

Reply:
@rubenwardy :eyes:

Reply:
This is no longer relevant, as I have changed the mainmenu to use an icon for del favourite. Take a look at the new design, I believe there's space for a pencil icon next to it

Reply:
I'm busy with other projects at the moment. Not planning on working on this anymore unless I get some spare time.

Reply:
The code has changed, & I am not very familiar with formspecs anymore.